### PR TITLE
chore: cmake user files to gitignore

### PR DESCRIPTION
**Pull request by sithlord48**
- Update `.gitignore` to also ignore common cmake user and generated files

### CODE REVIEW

**Review by nbolton — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,8 @@ deskflow-config.toml
 /scripts/*.egg-info
 /*.user
 *.ui.autosave
+
+#CMake ignores
+CMakeLists.txt.user
```
Comment (nbolton) [inline .gitignore:25]: Damn, I thought I added these already. Apparently not!

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# SPDX-FileCopyrightText: Deskflow Authors
```
Comment (nbolton) [inline .gitignore:1]: An alternative to "Deskflow Authors" (a bit vague) could be:
```suggestion
# SPDX-FileCopyrightText: Copyright (C) 2010 Jason Axelson <BostonVaulter@gmail.com>
# SPDX-FileCopyrightText: Copyright (C) 2011 Symless Ltd.
# SPDX-FileCopyrightText: Copyright (C) 2024 Chris Rizzitello
```

**Review by nbolton — DISMISSED** [review]

File: .gitignore (hunk 2 above)
Comment (sithlord48) [reply .gitignore:1]: newest goes to to the top but yeah ill do that

Comment (nbolton) [reply .gitignore]: Oh! Will remember that.

**Pull request merged**